Dockerfile

```diff
@@ -1,32 +1,25 @@
-FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.7
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6.9.1
 
 RUN yum clean all && \
     yum update -y && \
-    yum install -y git make gcc-c++ psmisc java-1.8.0-openjdk-devel && \
```

Comment:
The `ci_build.sh` script is working for me so I don't think this is needed here either. I think it is a hang over from dredd.